.github/workflows/swarco_itc2.yml

```diff
@@ -26,8 +26,9 @@ jobs:
       run: bundle install
     - name: Start ITC-2 simulator
       run: virsh -c qemu:///system start ts1001
-    - name: Wait for ITC-2 simulator to start
-      run: sleep 30s
```

Comment:
it would be great to replace the hardcoded 30s wait with something that waits until it's ready (but with a timeout)